components/fonts/font_cache_thread.rs

```diff
@@ -102,12 +103,19 @@ struct FontCache {
 }
 
 impl FontCache {
+    #[tracing::instrument(skip(self), fields(servo_profiling = true))]
```

Comment:
Sorry, this is maybe at bit late / the wrong place to ask, but what exactly does `fields(servo_profiling = true)` do? I'm asking because on OpenHarmony these events seem to also be logged via `hilog`, and it's kind of wierd seeing most log messages end with `; servo_profiling=true`. I've never seen it set to false, so I'm kind of wondering what it does.

Reply:
No worries! The idea behind adding the `fields(servo_profiling = true)` custom field is to help distinguish our own thread's profiling spans. Right now, we only send [our spans to hitrace](https://github.com/servo/servo/pull/33324), but if tracing ever merges spans from all threads, it could cause confusion. We could have used `name` instead of fields to differentiate them, but that would have made filtering a bit messier.

By adding the field, we can more easily separate our profiling spans from any other existing spans.

Reply:
I will add the PR to the MQ and if needed, we can discuss it further on Zulip